02_Methodiek.Rmd

```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
 
 Voorbeeld 2: in een rastercel zijn drie meetpunten gewenst. 
 Er zijn vijf peibuizen beschikbaar, twee met kwaliteitsvolle tijdreeksen, de drie andere met tijdreeksen van onvoldoende kwaliteit. 
 Deze rastercel zal behoren tot categorie 2 voor de twee peilbuizen van goede kwaliteit en tot categorie 1B: voor het derde meetpunt moet uit de drie peilbuizen met onvoldoende kwaliteit er één nog gekozen worden.
 
 
-[^1]: lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt.
+[^2]: lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt.
 
 De extreem droge zomers van 2017 en 2018 worden niet meegenomen in de berekening. Het is namelijk nu nog onduidelijk (door mogelijke naijling) of de actuele vegetatie geen nadelige effecten van deze twee zomers heeft ondervonden.
 
 Voor een meetpunt dat voldoet aan deze criteria kan een gemiddeld laagste grondwaterstand (GLG) berekend worden. 
 
 Op basis van de kwaliteit van de tijdreeksen kunnen dan per GT-groep de rastercellen in (één of meer van de volgende) vier categorieën opgedeeld worden:
 
-- **Cat. 1A**: de vraag is groter dan het aanbod: er is een gebrek aan meetpunten, aan de vraag kan enkel voldaan worden door nieuwe meetpunten te introduceren.
-- **Cat. 1B**: de vraag is groter aan het aanbod: er is een gebrek aan geschikte meetpunten, de vraag kan vervolledigd worden met actueel ongeschikte meetpunten
-- **Cat. 2**: de vraag is groter of gelijk aan het aanbod: er zijn geschikte meetpunten, juist voldoende of te weinig: elk meetpunt wordt geselecteerd en kan onmiddellijk ingezet worden.
-- **Cat. 3**: de vraag is kleiner dan het aanbod: er is een overschot aan geschikte meetpunten
+- **Cat. 1A**: de vraag is groter dan het aanbod, er is geen aanbod: aan de vraag kan enkel voldaan worden door nieuwe meetpunten te introduceren.
+- **Cat. 1B**: de vraag is groter dan het aanbod van peilbuizen van goede kwaliteit: er is een gebrek aan geschikte meetpunten, de vraag kan wel ingevuld worden met actueel ongeschikte meetpunten.
+Op basis van het aantal ongeschikte meetpunten, wordt nog een opdeling gemaakt:
+- **Cat. 1Ba**: het aantal ongeschikte meetpunten is onvoldoende tot juist voldoende om alle gewenste locaties te kunnen invullen.
+- **Cat. 1Bb**: het aantal ongeschikte meetpunten is groter dan het aantal gewenste locaties: er zal hiertussen nog een keuze moeten gemaakt worden.
+- **Cat. 2**: de vraag is groter of gelijk aan het aanbod van peilbuizen van goede kwaliteit: er zijn geschikte meetpunten, juist voldoende of te weinig: elk meetpunt wordt geselecteerd en kan onmiddellijk ingezet worden.
+- **Cat. 3**: de vraag is kleiner dan het aanbod van peilbuizen van goede kwaliteit: er is een overschot aan geschikte meetpunten
```

Comment:
duidelijk verwoord :+1: 
Wel blijft het voorlopig nog een mysterie waarom een rastercel tot meerdere categorieën zou kunnen behoren

Reply:
het begint me een beetje te dagen, na heen en weer kijken naar je twee voorbeelden. In feite ga je groepen van meetpunten binnen een rastercel groeperen, iets in de richting van:

- er zijn x goede
- er zijn y 'slechte'
- er zijn z - x goede tekort en z-x kan > of < zijn dan y, en z-x kan ook < 0

Je gaat, meen ik, vervolgens voor elk van deze _meetpuntgroepen_ (x, y, z-x), indien z-x verschillend van 0, één categorie toekennen (per rastercel). Dat zou misschien duidelijker verwoord moeten worden, het verklaart waarom een rastercel tot meerdere categorieën behoort (misschien is 'categorie' geen ideale term hier). Vooralsnog is dit een voorlopige suggestie, onder voorbehoud van verdere lezing.

Reply:
Zie nog [deze opmerking](https://github.com/inbo/n2khab-droogtemeetnet/pull/5#discussion_r344430340).

Reply:
Op basis van latere lezing (zie latere review), voeg ik hier een tentatieve definitie in op niveau van de rastercel, met behulp van x, y, z (z = totaal aantal nodig), en geënt op je bovenstaande beschrijving

- 1A: x < z en y = 0
- 1B: x < z en y > 0
  - 1Ba: x + y <= z
  - 1Bb: x + y > z
- 2: x <= z en y >= 0. Waarom de overlap met de voorgaande?
  - deze definitie is in overeenstemming met (regel 792 van Rmd 03): "_Als de kwaliteit van de tijdreeks van een aantal peilbuizen voldoende is om onmiddellijk in een meetnet te kunnen worden opgenomen, wordt de cel ook tot categorie 2 gerekend._" Echter: waarom deze complexiteit van iets tot 2 categorieën rekenen? De term 'categorieën' impliceert m.i. reeds 'wederzijds uitsluitend', dus geen overlap.
  - anderzijds, onder de kop 'het opleggen van bijkomende kwaliteitscriteria' (Rmd 03) staat dan weer: "_Er kan dan een groep cellen onderscheiden worden waarbij het gewenste aantal meetpunten juist overeenstemt met het aantal beschikbare peilbuizen die gelijkwaardig van kwaliteit zijn. Deze rastercellen behoren tot categorie 2._" Dat wijst op een definitie die veel logischer past in het rijtje: **x = z**. Dus: consequent maken. Als je 't mij vraagt, deze categorie beperkten tot x = z; overlap lijkt me onnodig complex.
- 3: x > z

_Het lijkt me een goed idee om, louter voor de uitleg, de peilbuizen van een GTgroep binnen een cel in klassen in te delen (goede, slechte, evt. meer, alleen als dat nodig is). Vervolgens de celcategorieën definiëren op basis van de onderlinge relatie tussen de frequentie van die klassen (cf. x, y hierboven) en op basis van de relatie met het gewenste aantal meetpunten (z hierboven) eenvoudige formules als hierboven._ Tekstueel is het m.i. moeilijker te volgen dan met een formulevorm. Zit daar iets in?

Voorts, als volgens de definities x = z en x > z geen cellen tot cat 2 en 3 behoren, dan kan dit bij de uitwerking zo gesteld worden en hoef je alleen verder met 1A en 1B (wat je alleszins doet), zonder die laatste twee verder in relatie te brengen met 2 en 3. (In het andere geval: dan was me niet duidelijk dat dit zo is).

Zie ook een aantal opmerkingen bij de review in PR #3 .